src/test_helper.py

```diff
@@ -0,0 +1,57 @@
+#! /usr/bin/env python
+"""Dummy docstring."""
+from test_modules import checking_answers, picking_random_elements, saving_into_file, adding_new_questions
+
+questions = ["Ile to 2+2: ", "Jaka jest wartość liczby Pi: ", "Czy słoń to ssak: "]
```

Comment:
Constants should be defined with upper case.
```suggestion
QUESTIONS = ["Ile to 2+2: ", "Jaka jest wartość liczby Pi: ", "Czy słoń to ssak: "]
```